server/data/personClient.ts

```diff
@@ -66,6 +67,16 @@ export default class PersonClient {
     return response as Array<ActiveOffence>
   }
 
+  async spaceBookings(crn: string, includeCancelled = false): Promise<Array<Cas1SpaceBooking>> {
```

Comment:
Does this API call return a `Cas1SpaceBooking` ? I thought Venkatesh said it was a `Cas1SpaceBookingShortSummary` This has got to be correct or all the other type checking is invalid.